src/Catalyst.Abstractions/Kvm/Models/BlockForRpc.cs

```diff
@@ -34,50 +35,51 @@
 
 namespace Catalyst.Abstractions.Kvm.Models
 {
+    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
     public class BlockForRpc
     {
-        private readonly BlockDecoder _blockDecoder = new BlockDecoder();
+        // private readonly BlockDecoder _blockDecoder = new BlockDecoder();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)